cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/log/LogHandlerTestRun.java

```diff
@@ -462,11 +492,22 @@ private void verifyLogs(HttpResponse response, String entityId, String format, b
     }
   }
 
+  private void verifyFieldsToSuppress(LogData logData) {
+    if (logData.getLogLevel() != null || logData.getLineNumber() != 0) {
```

Comment:
Should `logData.getLineNumber()` be null or 0? Can you change the type of `lineNumber` to `Integer`?